mzLib/UsefulProteomicsDatabases/ProteinDbWriter.cs

```diff
@@ -117,10 +117,14 @@ public class ProteinDbWriter
                         }
                         writer.WriteEndElement();
                     }
-                    foreach (var proteolysisProduct in protein.ProteolysisProducts)
+
+                    //for now we are not going to write top-down biomarkers generated for top-down biomarker search. 
```

Comment:
Could you add a note that biomarker-specific type information is included after `(`, so that's clearer for posterity?